graphql_compiler/tests/test_helpers.py

```diff
@@ -744,7 +784,11 @@ def generate_schema_graph(orientdb_client):
     return get_orientdb_schema_graph(schema_data, index_query_data)
 
 
-def generate_schema(orientdb_client, class_to_field_type_overrides=None, hidden_classes=None):
+def generate_schema(
+    orientdb_client: OrientDB,
+    class_to_field_type_overrides: Optional[Dict[str, Dict[str, GraphQLSchemaFieldType]]] = None,
```

Comment:
I think we reference `class_to_field_type_overrides` a few times in the code. Given `Optional[Dict[str, Dict[str, GraphQLSchemaFieldType]]]` is quite lengthy, too, I would consider storing the whole annotation so it can be re-used.